src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/watasks/controllers/GetTaskByIdControllerTest.java

```diff
@@ -363,4 +363,127 @@ public void should_replace_additional_properties_in_configuration_dmn_and_return
         common.cleanUpTask(taskId);
     }
 
+    @Test
+    public void should_return_a_200_with_task_id_in_description_when_creating_standalone_task() {
+        String taskType = "reviewSpecificAccessRequestLegalOps";
+        String taskName = "review specific access request legal ops";
+        TestVariables taskVariables = common.setupWAStandaloneTaskAndRetrieveIds(
```

Comment:
Is this just checking in Camunda?

Reply:
no it is not, it is checking CFT db, in Camunda, the task Id is the same as the task type